packages/discovery-provider/src/api/v1/users.py

```diff
@@ -2604,23 +2605,38 @@ def get(self, id, authed_user_id):
 
 
 sales_download_parser = current_user_parser.copy()
+sales_download_parser.add_argument(
```

Comment:
added for backward compatibility sake while we transition to generating csv on FE